ibctesting/utils_test.go

```diff
@@ -72,7 +72,21 @@ func (suite *IBCTestUtilSuite) CreateRollapp() {
 		suite.rollappChain.ChainID,
 		10,
 		[]string{},
-		nil,
+		[]rollapptypes.TokenMetadata{
+			{
+				Description: "stake as rollapp token",
+				DenomUnits: []*rollapptypes.DenomUnit{
+					{
+						Denom:    "arax",
+						Exponent: 0,
```

Comment:
I'd create here a fuller object with more  denom units type to check a real use case.